bower.json

```diff
@@ -2,34 +2,35 @@
   "name": "origin-web-console",
   "version": "1.3.0-alpha",
   "dependencies": {
-    "angular": "1.3.20",
+    "angular": "1.5.11",
     "json3": "3.3.2",
     "es5-shim": "3.1.1",
-    "angular-resource": "1.3.20",
-    "angular-cookies": "1.3.20",
-    "angular-sanitize": "1.3.20",
-    "angular-animate": "1.3.20",
-    "angular-touch": "1.3.20",
-    "angular-route": "1.3.20",
+    "es5-dom-shim": "*",
```

Comment:
Yeah, I'm fine with it, although something we might look at. Maybe we can just exclude (not for this PR)